newIDE/app/src/ResourcesList/index.js

```diff
@@ -31,7 +31,7 @@ const styles = {
 type State = {|
   renamedResource: ?gdResource,
   searchText: string,
-  resourcesWithMissingPath: { [string]: boolean },
+  resourcesWithErrors: { [string]: 0 | 1 | 2 },
```

Comment:
While it's a good idea to use an "enum" like this, it would be clearer I think with strings:

`resourcesWithErrors: { [string]: "no-error" | "not-found" | "outside-project-foler" }`

You'll have to change the rest of the references to 0, 1 and 2, but this should be clearer :)